### ISSUE TITLE
Don't use localizeddate to avoid Intl errors

### ISSUE THREAD

**Issue by WouterJ** [symfony/symfony-demo#329]
I didn't have a supported timezone setting in my PHP.ini, resulting in the following exception:

 > An exception has been thrown during the rendering of a template ("Time zone different than GMT or UTC is not supported as a formatting output. Please install the "intl" extension for full localization capabilities.") in base.html.twig at line 150.

I think this will be a common issue for  Windows users. As this demo should work straightaway, I propose to remove `|localizeddate` from the templates and simply replace them by `|date('r')` or the like.

**Comment by javiereguiluz** [comment]
I don't agree with this proposal. Having a timezone properly configured is one of the few mandatory Symfony's requirements: http://symfony.com/doc/current/reference/requirements.html

**Comment by WouterJ** [comment]
@javiereguiluz there is a difference between properly configuring a timezone and configuring a timezone supported by the Symfony Intl component (used when the intl extension is not available, which is the case for Windows PHP).

I've set my datetime setting to `Europe/Amsterdam`, which is perfectly correct. However, the Symfony Intl component supports only `GMT` or `UTC`.

If it's mandatory to configure the timezone with either GMT or UTC when using Symfony, the Symfony Requirements have to be updated. At the moment, I get this output when running the requirement checker (there is no note about timezone not being GMT or UTC):
```
C:\wamp\www\insite-admin>php app/check.php

Symfony Requirements Checker
~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> PHP is using the following php.ini file:
  C:\wamp\bin\php\php7.0.2\php.ini

> Checking Symfony requirements:
  ..............................W.W.......


 [OK]
 Your system is ready to run Symfony projects


Optional recommendations to improve your setup
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

 * intl ICU version installed on your system (56.1) should match the
   ICU data bundled with Symfony (55.1)
   > In most cases you should be fine, but please verify there is no
   > inconsistencies between data provided by Symfony and the intl
   > extension. See https://github.com/symfony/symfony/issues/15007
   > for an example of inconsistencies you might run into.

 * a PHP accelerator should be installed
   > Install and/or enable a PHP accelerator (highly recommended).


Note  The command console could use a different php.ini file
~~~~  than the one used with your web server. To be on the
      safe side, please check the requirements from your web
      server using the web/config.php script.
```

**Comment by javiereguiluz** [comment]
I don't know what to do about this. @WouterJ is right and this app should work out-of-the-box. But on the other hand, removing `localizeddate` makes the application worse and removes an interesting feature to show to developers.

What can we do to maintain `localizeddate` but avoid errors on Windows?

**Comment by xabbuh** [comment]
We could ship a custom `date` filter that checks which version to use.

**Comment by tcicit** [comment]
I'm new to symphony and i got the samme error. So what is the solution? I have no Linux only Windows.

**Comment by WouterJ** [comment]
Either enable intl for windows (by downloading the intl ddl files and enabling it in php.ini). this is rather complex though.

The other solution is to look for `|localizeddate` in the `app/Resources/views` directory and replace it with `|date('r')`

**Comment by mtsanford** [comment]
I'm finding this frustrating as well.    So many hoops to jump through to get intl php extension installed on a Mac.    After an hour I finally realize I have to disable the System Integrity Protection system so the build system can modify /usr, but can't even do that since my non-Apple keyboard won't let me boot into recovery mode :(, so I guess I'll try rewriting the templates.

**Comment by javiereguiluz** [comment]
I'm trying to fix this problem in #335.

**Comment by wafae1** [comment]
In most cases you should be fine, but please verify there is no inconsistencies between data provided by Symfony and the intl extension. See https://github.com/symfony/symfony/issues/15007 for an example of inconsistencies you might run into.

**Comment by Baptistech** [comment]
do a sudo apt-get install php5-intl

**Comment by javiereguiluz** [comment]
@Baptistech you are right ... but that won't be useful for our friends using Windows.

**Comment by Julix89able** [comment]
I'm using Xampp and Symfony and I was able to fix the problem due to change the php.ini file in the xampp/php directory. You just have to delete the ; in the line 883 there has to be this:
extension=php_intl.dll
Before it was:
;extension=php_intl.dll

**Comment by andig** [comment]
+1 for making this work without ext-intl. Sofar symfony (demo) is the only application I've encountered requiring this extension. Would a polyfill help?

**Comment by romqin** [comment]
>@javiereguiluz :  this app should work out-of-the-box

What about adding the Docker support ? This would solve this problem.

**Comment by andig** [comment]
> What about adding the Docker support ? This would solve this problem.

Not necessarily available on Windows either, much less non-admin accounts.

**Comment by samayo** [comment]
This is very problematic issue on CentOs 7.3 which has only ICU version 50.1 
There is no way to easily get past the Symfony configuration page, let alone create anything with it.

**Comment by wouterj** [comment]
@samayo by default, Symfony doesn't depend on the intl extension/ICU data version. Please check if you're using a third party library causing this error and try fixing it in their codebase/disabling the bundle/library.

*Btw, this is a support related question. The GitHub repositories are purely managed to discuss new features/report bugs. Please use one of the channels mentioned in http://symfony.com/support or [Slack](https://slackinvite.me/to/symfony-devs)*

**Comment by samayo** [comment]
@wouterj  Thanks for the links. But I am not using any third library date. I set up CentOs7.3 on a VirtualBox, I downloaded Nginx, PHP-FPM and went to the Symfony page and tried to download it, with the `symfony new blog 3.0` and I start the built-in server and header over to `localhost:8000/config.php` and there it is the error. it never goes away. It says, I have ICU version 50.1 but Symfony needs version 57.1 

Stuck with this for the third day in a row.
